components/playback/controls/PlateBarcode.vue

```diff
@@ -24,13 +24,14 @@
 <script>
 import { mapState } from "vuex";
 import playback_module from "@/store/modules/playback";
-
+import { TextValidation } from "@/js_utils/text_validation.js";
 /**
  * @vue-data {String} platebarcode - Current plate bar code
  * @vue-data {String} playback_state_enums - Current state of playback
  * @vue-computed {String} playback_state - Current value in Vuex store
  * @vue-event {String} validatePlateBarcode - User entered String parser
  */
+const TextValidation_platebarcode = new TextValidation("platebarcode");
```

Comment:
`platebarcode` is two words.  since it is being passed as a string, we have a variety of options, we could use snake_case, kebab-case, or an actual space.  But we should never just push two words together as it makes it hard to read.

Since this is a string and it could accept a space, I would propose that we use that as I think it is the most readable.  But if you think snake_case would be better you could choose that also

Reply:
Agreed, renamed as `plate_barcode`